### ISSUE TITLE
Load image from link

### ISSUE THREAD

**Issue by phucdth12a** [tungvoduc/DTPhotoViewerController#31]
```
func photoViewerController(_ photoViewerController: DTPhotoViewerController, configurePhotoAt index: Int, withImageView imageView: UIImageView) {
    imageView.pin_setImage(from: images[index].link?.toURL, [placeholderImage:](url) cell.imageView.image)
    }
  }
```

when loading image i zoom imageView and when loaded image this is change zoomscale

**Comment by tungvoduc** [comment]
@phucdth12a I will make a fix to disable zooming if imageView does not have an image.

**Comment by tungvoduc** [comment]
@phucdth12a I've fixed it in version `1.2.5`. Please updat your pod.

**Comment by tungvoduc** [comment]
#32
